### ISSUE TITLE
Documentation on sw-controlled interface for flushing cache blocks

### ISSUE THREAD

**Issue by OrkunAliOzkan** [chipsalliance/rocket-chip-inclusive-cache#23]
Hi, 

I have integrated the L2 cache into the rocketchip provided by chipsalliance, and I would like to run a cache flush, but I do not know how to invoke this, would anyone have any related .asm showing the command in use or reference to any docs?

Thanks :)

**Comment by rralf** [comment]
Yep, this is what I'd like to know as well. Were you able to find some documentation?

**Comment by jerryz123** [comment]
Unfortunately not, but the interface is not complex.Writing physical addresses into the flush64 memory-mapped register is all that is necessary.

**Comment by rralf** [comment]
Thanks a bunch Jerry. I just found the Control.scala that helped a lot. Just a question: Besides the flush64 command, is there a way to invalidate the cache? I need it for a benchmark.

**Comment by rralf** [comment]
Ah damn, I already assumed that. I'll try to implement it.
